lib/mini_sql/builder.rb

```diff
@@ -11,6 +11,13 @@ def initialize(connection, template)
     @is_prepared = false
   end
 
+  def inject_sql(injects)
```

Comment:
I think that `inject_sql` is not the best name, https://en.wikipedia.org/wiki/SQL_injection the term is already well known, maybe there are other variants of the name?

Reply:
maybe `sql_literal`, `sql_section`

Reply:
I like `sql_literal`